report/pass_report.md

```diff
@@ -1,7 +1,13 @@
 ## Armstrong Test Passed
 
+__This file is automatically generated, please do not edit it directly.__
+
 ### Tested resource types and addresses
```

Comment:
Can we add a summary at the beginning, this summary should contain a short description of what errors exist in current report, like:

1. How many errors of resource types
2. Coverage status

After that, we can add a description like "The following are more details"

Reply:
> 1.How many errors of resource types

if there are any error for the resource types, it will not show in the pass report, but will show in the error report

> 2. Coverage status

a coverage summary is added now.